index.js

```diff
@@ -39,6 +86,30 @@ app.get("/messages", (req, res) => {
   res.json(messages.slice(afterId));
```

Comment:
Now that we can deleting things we can't slice the messages like this anymore. We need to find the index of the message with message.id === afterId and then slice based on that index